docs/faq.md

```diff
@@ -0,0 +1,53 @@
+---
+id: faq
+title: Frequently Asked Questions
+sidebar_label: FAQ - Questions
+---
+
+## What is the recommended single-spa setup?
+We recommend a setup that uses ES modules + [import maps](https://github.com/WICG/import-maps) (or [SystemJS](https://github.com/systemjs/systemjs) to polyfill these if you need better browser support).  This setup has several advantages:
+1. Common libraries are easy to manage, and are only downloaded once. You can also preload these for a small speed boost as well using the standard preload spec.
+2. Sharing code / functions / variables is as easy as an import statement, just like in a monothlithic setup
```

Comment:
typo: `monolithic`

Reply:
knew I could count on you to find that ;)